policyengine_us/parameters/gov/states/oh/tax/income/deductions/unreimbursed_medical_care_expenses/rate.yaml

```diff
@@ -0,0 +1,15 @@
+description: Ohio deducts medical care expenses in excess of this fraction of federal adjusted gross income.
+values:
+  2021-01-01: 0.075
+metadata:
+  unit: /1
+  label: Ohio Unreimbursed Medical Care Expenses Deduction AGI threshold
+  reference:
+    - title: Ohio Tax Form 2021 - Unreimbursed Medical Care Expenses Worksheet (Ohio Schedule of Adjustments, Line 36)
+      href: https://tax.ohio.gov/static/forms/ohio_individual/individual/2021/pit-it1040-booklet.pdf#page=18
+    - title: Ohio Tax Form - Deductions Line 36 – Unreimbursed Medical & Health Care Expenses
+      href: https://tax.ohio.gov/static/forms/ohio_individual/individual/2022/it1040-sd100-instruction-booklet.pdf#page=18
+    - title: Ohio Tax Form - Unreimbursed Medical Care Expenses Worksheet (Ohio Schedule of Adjustments, Line 36)
+      href: https://tax.ohio.gov/static/forms/ohio_individual/individual/2022/it1040-sd100-instruction-booklet.pdf#page=27
+    - title: Ohio Revised Code Section 5747.01(A)(10)(b)
```

Comment:
use this link instead: https://casetext.com/statute/ohio-revised-code/title-57-taxation/chapter-5747-income-tax/section-574701-income-tax-definitions

Reply:
for legal code